pom.xml

```diff
@@ -25,7 +25,7 @@
   <parent>
     <groupId>com.xwiki.parent</groupId>
     <artifactId>xwikisas-parent-platform</artifactId>
-    <version>14.10</version>
+    <version>14.10-1</version>
```

Comment:
This does not seem related to this PR. If the wrong parent was used by mistake, you can have a separate commit on the master branch with the miscellaneous annotation used, e.g. :
```
[misc] Use the correct XWiki parent 
* use the 14.10-1 xwikisas parent in order to have XWiki 14.10 as the parent, since the 14.10 parent will actually bring XWiki version 14.10.2
```